packages/web-components/src/calendar/calendar.ts

```diff
@@ -404,7 +466,7 @@ export class Calendar extends FASTCalendar {
         this.selectedDates += `${month}-${day}-${year},`;
       }
     } else {
-      this.selectedDates = `${month}-${day}-${year},`;
+      this.selectedDates = `${month}-${day}-${year}`;
```

Comment:
M/D/Y vs Y/M/D
Is there a reason to use both or just one of these? Looking below at line 552 using Y/M/D

Reply:
The `M-D-Y` formatting is to align with how the attribute `selected` is added to the day cells in the base FAST component. The logic for this in the [FASTCalendar](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/calendar/calendar.ts) `calendar.ts` file is the lines in the screenshot below
![image](https://github.com/microsoft/fluentui/assets/65481565/7e7f5b71-32e9-47e7-9a8e-d27dd4f9dea4)

While that format works internally for just checking a string against the selected dates string in the component class, the format isn't supported by Firefox, so that's why I'm using the supported `Y/M/D` format for when the browser needs to interpret the navigated date.